templates/example_neighborhood_analysis_script.ipynb

```diff
@@ -268,9 +300,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.10"
+   "version": "3.8.5"
```

Comment:
Can you try running the notebook with Python 3.6? I don't think anything will change but since our repo uses Python 3.6, it would be good to make sure this code runs on that version just in case.

Reply:
Checked it now, it works as expected